### PR TITLE
[BPK-1041] Add focused prop to native card component

### PR DESCRIPTION
**Pull request by shaundon**
*Note:* The ticket referred to `focussed` but I decided to use `focused` for the prop name, as it's more common to use this in U.S. English.

screen shot 2017-10-19 at 17 47 13

screenshot_1508483386

### CODE REVIEW

**Comment by k0nserv** [comment]
Missing changelog entry other than that it loosk good

**Comment by shaundon** [comment]
@k0nserv damn, knew I'd forget something!

**Comment by shaundon** [comment]
Now added changelog.

**Comment by jamesf3rguson** [comment]
LGTM :)

File: native/packages/react-native-bpk-component-card/readme.md [hunk 1]
```diff
@@ -51,5 +52,6 @@ export default class App extends Component {
 | Property            | PropType  | Required | Default Value |
 | -----------         | --------- | -------- | ------------- |
 | children            | node      | true     | -             |
+| focused             | boolean   | false    | false          |
```
Comment (matteo-hertel) [inline native/packages/react-native-bpk-component-card/readme.md]: last pipe is not aligned 😄

**Pull request merged**